### PR TITLE
Avoid displaying just title in breadcrumbs

### PR DESCRIPTION
**Pull request by nineteendo**
See https://github.com/pradyunsg/furo/pull/843

### CODE REVIEW

**Comment by pradyunsg** [comment]
Hmm... Isn't `breadcrumb_include_page` sufficient for controlling whether just the page title is shown?

**Comment by pradyunsg** [comment]
From https://sphinx-basic-ng.readthedocs.io/en/latest/usage/components/breadcrumbs/:

> There are 2 values that can be provided via the html-context, which provide control over the components shown in the breadcrumbs:
>
> [snip]
> - breadcrumb_include_page: If truthy, the current page is included in the breadcrumb as the final list-item, with only a span (there’s no a[href] in it). The corresponding list-item will have an additional class: breadcrumb-nav-page-item.

**Comment by nineteendo** [comment]
It's just a bit pointless to show breadcrumbs if there are no links to parent pages. You're just repeating the title for no benefit. By not doing this you can use the style of https://just-the-docs.com.

**Comment by pradyunsg** [comment]
> It's just a bit pointless to show breadcrumbs if there are no links to parent pages.

I agree, but disagree that having it in the DOM is useless.

Scroll on https://pradyunsg-cpython-lutra-testing.readthedocs.io/en/latest/c-api/index.html -- that's an excellent example of why having the current page title in breadcrumbs is useful.

**Comment by nineteendo** [comment]
I'm afraid you don't understand what I mean, so let me try again with some images:

1. The breadcrumbs here don't add any value, they just repeat the title without linking to a parent:

Screenshot 2024-10-06 at 13 29 13

---

2. They also don't add value if they link to the home page on the home page (that's why we have `pagename != 'index'`):

Screenshot 2024-10-06 at 13 38 34

---

3. On child pages, they're useful because they link to the parents:

Screenshot 2024-10-06 at 13 31 48

**Comment by nineteendo** [comment]
Do you think this should be configurable? In the long run I want to delete my fork.

**Comment by nineteendo** [comment]
@pradyunsg, did you forget about this?

**Comment by pradyunsg** [comment]
No, but I am gonna close this out since I know that this is something that it's useful to provide (see the Lutra example earlier on mobile - you need to scroll down for it).

For themes that don't want this, they can remove it by using the boolean available for this functionality.

**Comment by nineteendo** [comment]
Note that `breadcrumb_include_page` controls 2 things:

- Forcing the breadcrumbs to be present
- Including the current page in the breadcrumbs

I can't disable the first behaviour.

**Comment by nineteendo** [comment]
Could you re-open this? I want to make this configurable with `breadcrumb_top_level`.

**Comment by nineteendo** [comment]
@pradyunsg
